Formula/librtlsdr.rb

```diff
@@ -23,9 +24,36 @@ class Librtlsdr < Formula
   depends_on "pkg-config" => :build
   depends_on "libusb"
 
+  # Upstream has no release since 0.6.0 so we use the Debian patchset:
+  # https://packages.debian.org/bullseye/rtl-sdr
+  patch do
+    url "https://deb.debian.org/debian/pool/main/r/rtl-sdr/rtl-sdr_0.6.0-3.debian.tar.xz"
+    sha256 "6cb04ec046df03a3a47a0bbf1162dffa20ccd36b54d1d237e0ef793178f8b5b1"
+    # Apply patches listed in patches/series
+    apply %w[
+      patches/0002-lib-Add-workaround-for-Linux-usbfs-mmap-bug.patch
+      patches/0003-contrib-jenkins.sh-run-make-maintainer-clean.patch
+      patches/0004-lib-fix-memory-leak-in-rtlsdr_open.patch
+      patches/0005-lib-disable-usbfs-zero-copy-support-by-default.patch
+      patches/0006-rtl_eeprom-fix-warnings.patch
+      patches/0007-Fix-building-librtlsdr-on-OpenBSD.patch
+      patches/0008-lib-Add-GPIO-version-of-the-bias-tee-configuration-A.patch
+      patches/0009-rtl_biast-Add-rtl_biast.patch
+      patches/0010-allow-building-librtlsdr-as-CMake-subproject.patch
+      patches/0011-Fixed-issues-compiling-on-Windows-with-MSVC-CMake-an.patch
+      patches/0012-set-CMake-policy-CMP0075-if-it-exists.patch
+      patches/0013-lib-enable-better-UHF-reception-862MHz-for-FC0013.patch
+      patches/better-user-input-handling
+      patches/use-udev-access-rules
+      patches/add-ipv6-support
+      patches/improve-librtlsdr-pc-file
+      patches/modernize-cmake
+    ]
+  end
+
   def install
     mkdir "build" do
-      system "cmake", "..", *std_cmake_args
+      system "cmake", "..", *std_cmake_args, "-DCMAKE_INSTALL_NAME_DIR=#{lib}"
```

Comment:
Why do we need `CMAKE_INSTALL_NAME_DIR`? Why not `CMAKE_INSTALL_RPATH`?

Reply:
I found some formulae use `CMAKE_INSTALL_NAME_DIR` and the compiled binary's shared library path is the same absolute path as previous. Do you recommend using `-DCMAKE_INSTALL_RPATH=#{rpath}`?

Reply:
> Do you recommend using `-DCMAKE_INSTALL_RPATH=#{rpath}`?

Yes, as long as what `rpath` is correct. Using `rpath` requires no relocation when using bottles, and more formulae use it:

```
❯ rg 'CMAKE_INSTALL_RPATH' -l | wc -l
116
❯ rg 'CMAKE_INSTALL_NAME_DIR' -l | wc -l
12
```

Reply:
Thanks for you advice, pushed a new commit.